### DISCUSSION TITLE
Start new plot when copy last plot to Final Directory?

### DISCUSSION THREAD

**Discussion by berlef80** [madMAx43v3r/chia-plotter#572] [Q&A]
Could someone clarify...during the last step when the finished plot is copied to its final directory destination, does the next plot already start being created, or does it wait until the plot is copied and then start with the new plot?
Thanks!

**Comment by SebMoore** [comment]
> Could someone clarify...during the last step when the finished plot is copied to its final directory destination, does the next plot already start being created, or does it wait until the plot is copied and then start with the new plot?
> Thanks!

It starts immediately and runs the copy in the background
